storage/simulation/script.go

```diff
@@ -282,6 +301,15 @@ func (s *Script) parse(scriptFile string) error {
 			}
 		}
 
+		// Get the percentage value.
+		if len(elements) > 5 {
+			if details.percent, err = strconv.Atoi(elements[5]); err != nil {
+				return util.Errorf("line %d PERCENT could not be parsed: %s", lineNumber, elements[5])
+			}
```

Comment:
I would validate the range of `percent` here and return an error if it's out of bounds instead of silently clamping it above. We may want to treat a value of 0 as an error too since such a line will have no effect (and it may indicate a missing/extra value in the line)

I also have a personal preference for converting percentages into a float between 0 and 1 and doing all probability calculations that way, but it doesn't really make any difference here.